src/Azure.Functions.Cli/Common/Constants.cs

```diff
@@ -25,11 +25,13 @@ internal static class Constants
         public const string FunctionsExtensionVersion = "FUNCTIONS_EXTENSION_VERSION";
         public const string StorageEmulatorConnectionString = "UseDevelopmentStorage=true";
         public const string AzureWebJobsStorage = "AzureWebJobsStorage";
+        public const string PackageReferenceElementName = "PackageReference";
+        public const string PackageReferenceVersionElementName = "Version";
 
         public static string CliVersion => typeof(Constants).GetTypeInfo().Assembly.GetName().Version.ToString(3);
         public static string CliBetaRevision => typeof(Constants).GetTypeInfo().Assembly.GetName().Version.MinorRevision.ToString();
```

Comment:
is this still valid?

Reply:
only `PackageReferenceElementName` is needed, `PackageReferenceVersionElementName` is not used